docs/matrix-core/examples/jstests.md

```diff
@@ -22,7 +22,7 @@ node -v
 ```
 
 ## Download and Prepare CORE
-```bash
+```language-bash
 git clone https://github.com/matrix-io/matrix-creator-malos;
 cd matrix-creator-malos;
 git submodule update --init;
```

Comment:
It is outdated, check: now malos doesn't have a submodule.

Reply:
removed 
```
 git submodule update --init;
```